Engine/DataFeeds/BacktestingOptionChainProvider.cs

```diff
@@ -33,22 +33,32 @@ public class BacktestingOptionChainProvider : IOptionChainProvider
         /// <param name="symbol">The underlying symbol</param>
         /// <param name="date">The date for which to request the option chain (only used in backtesting)</param>
         /// <returns>The list of option contracts</returns>
-        public IEnumerable<Symbol> GetOptionContractList(Symbol symbol, DateTime date)
+        public IEnumerable<Symbol> GetOptionContractList(Symbol underlyingSymbol, DateTime date)
         {
-            if (symbol.SecurityType != SecurityType.Equity)
-            {
-                throw new NotSupportedException($"BacktestingOptionChainProvider.GetOptionContractList(): SecurityType.Equity is expected but was {symbol.SecurityType}");
```

Comment:
Think we might keep this check but just add Futures also? I can see a user trying to send in an option for example?